yarn-project/blob-sink/src/client/http.ts

```diff
@@ -32,6 +40,65 @@ export class HttpBlobSinkClient implements BlobSinkClientInterface {
     };
   }
 
+  public async testSources() {
```

Comment:
Should this function count how many sources are reachable and throw if that number is 0 by the end of the function? Not much point in keeping the archiver up if it can't get blobs it needs to synch

Reply:
I like the idea!